python-sdk/nuscenes/eval/detection/README.md

```diff
@@ -11,44 +11,53 @@ In this document we present the rules, results format, classes, evaluation metri
 - [Evaluation metrics](#evaluation-metrics)
 - [Leaderboard](#leaderboard)
 
-
 ## Introduction
 Here we define the 3D object detection task on nuScenes.
-The goal of this task is to place a 3D bounding box around 10 different categories of object,
+The goal of this task is to place a 3D bounding box around 10 different object categories,
 as well as estimating a set of attributes and the current velocity vector. 
 
-This document outlines rules, details, and metrics for the task.
-
 ## Challenges
 ### Workshop on Autonomous Driving, CVPR 2019
 The first nuScenes detection challenge will be held at CVPR 2019.
 Submission window opens in April 2019 and closes June 15th.
 Results and winners will be announced at the Workshop on Autonomous Driving ([WAD](https://sites.google.com/view/wad2019)) at [CVPR 2019](http://cvpr2019.thecvf.com/).
 
-
-## General rules
+## Submission rules
 * We release annotations for the train and val set, but not for the test set.
 * We release sensor data for train, val and test set.
-* The maximum time window of past sensor data that may be used is 0.5s.
 * Users make predictions on the test set and submit the results to our eval. server, which returns the metrics listed below.
 * We do not use strata. Instead, we filter annotations and predictions beyond class specific distances.
-* Every submission provides method information We encourage publishing code, but do not make it a requirement.
+* The maximum time window of past sensor data and ego poses that may be used at inference time is 0.5s (<= 10 lidar sweeps, <= 6 camera images). At training time there are no restrictions.
```

Comment:
Two comments
- Suggestion to add for clarity: _"No state should be maintained between detections. The predictions on each sample should be independent of the previous prediction."_
- 0.5s is not always equal to <=10 LIDAR sweeps due to slight irregularities in sampling frequency. I'm OK if we allow max 10 LIDAR sweeps, but it could technically be a violation of the 0.5 second rule.

Reply:
Updated 2) and ignored 1) as discussed.